js/app.js

```diff
@@ -0,0 +1,38 @@
+$(function() {
+  // 1. listen for a click event
+
+  $("#stopButton").click(redLight);
+  $("#slowButton").click(yellowLight);
+  $("#goButton").click(greenLight);
+
+
+  function redLight() {
+    clearLights();
+    $("#stopLight").css("background-color", "red");
+    $("#bulb").css("backgroundColor", "black");
+   
+  }
+
+
+  function yellowLight() {
+    clearLights();
+    $("#slowLight").css("background-color", "yellow");
+    $("#bulb").css("backgroundColor", "black");
+    
+  }  
+
+  function greenLight() {
+    clearLights();
+    $("#goLight").css("background-color", "green");
+    $("#bulb").css("backgroundColor", "black");
+  
+  }
+
+  function clearLights() {
+    $("#stopLight").css("background-color" , "black");
```

Comment:
@Schultisal the selector culd have been `$('.bulb').css("background-color" , "black");`. This would apply to all the lights since they all have the same bulb class.